buildtest/cli/helpcolor.py

```diff
@@ -0,0 +1,15 @@
+from buildtest.defaults import console
+from rich.color import ANSI_COLOR_NAMES
+from rich.table import Table
+
+
+def print_available_colors():
+    """Print the available color options in a table format on background of the color option."""
+    table = Table(
+        "[blue]Number",
+        "[blue]Color",

```

Comment:
you can replace the `[blue]` with argument `header_style='blue'`. See https://rich.readthedocs.io/en/stable/reference/table.html#rich.table.Table. I believe this will color the headers based on the color. Try it out i believe it should work

Reply:
Done !